src/Build/Evaluation/LazyItemEvaluator.LazyItemOperation.cs

```diff
@@ -51,12 +53,14 @@ protected LazyItemOperation(OperationBuilder builder, LazyItemEvaluator<P, I, M,
 
             public virtual void Apply(ImmutableList<ItemData>.Builder listBuilder, ImmutableHashSet<string> globsToIgnore)
             {
+                MSBuildEventSource.Log.ApplyStart();
```

Comment:
Name this something like `ApplyLazyItemOperations` and pass at least `_itemElement.ItemType`. The existing evaluation profiler tracks path and line number, too, which could be very nice.